dashboard/config/locales/en.yml

```diff
@@ -23,7 +23,9 @@ en:
   errors:
     messages:
       not_saved: 'Please correct the following errors:'
-      too_young: 'Your teacher must accept the updated Terms of Service before you can access this content. Your teacher can do this by logging into Code Studio and clicking the "Review Updated Terms" button near the top of Code Studio homepage.'
+      too_young: 'This content has age restrictions in place and is not available for younger students.'
+      teacher_must_accept_terms: 'Your teacher must accept the updated Terms of Service before you can access this content. Your teacher can do this by logging into Code Studio and clicking the "Review updated terms" button near the top of the Code Studio homepage.'
```

Comment:
Discussed with @poorvasingal offline. Yes, the intention was for students to probably show teachers the error message. Teachers may ignore the interstitial when it pops up and not realize that it gives students access to content, so this error message gives them more information on how it's related. I'll rephrase it a bit, but it at least gives a little more helpful information to the teacher. We'll also watch the data for what % of teachers don't accept the latest terms and we can do more to try and increase it like making the banner more prominent or showing the interstitial on more pages.